harness/determined/cli/master.py

```diff
@@ -125,6 +155,31 @@ def logs(args: argparse.Namespace) -> None:
                 help="number of lines to show, counting from the end "
                 "of the log (default is all)")
         ]),
+        cli.Cmd("cluster-message", None, "set or clear cluster-wide message", [
+            cli.Cmd("set", set_message, "create or edit the displayed cluster-wide message", [
+                cli.Arg("-s", "--start", default=datetime.datetime.utcnow().strftime(timestring),
```

Comment:
should be able to just do `datetime.datetime.utcnow().isoformat("T") + "Z"`. we do this in other places in the python, and i'd rather not have separate time fmt strings lying around if possible.

eventually we should centralize/standardize timestamps used across the codebase, or maybe add a method in `determined.common.util` for `utc_now` or something.

Reply:
Oh, that's great. Thanks!